src/mots/cli.py

```diff
@@ -156,6 +157,46 @@ def export(args: argparse.Namespace) -> None:
             f.write(output)
 
 
+def write(args: argparse.Namespace):
+    """Set a specified settings variable to the provided value."""
+    key = args.key[0]
+    value = args.value[0]
+    with settings.OVERRIDES_FILE.open("r") as f:
+        overrides = yaml.load(f) or {}
+
+    if key not in settings.DEFAULTS:
+        raise ValueError(f"{key} does not exist in defaults.")
+
+    _type = type(settings.DEFAULTS[key])
+
+    if _type(value) == overrides.get(key):
+        raise ValueError(f"{key} is already set to {value} in overrides file.")
+
+    overrides[key] = _type(value)
+    print(f"{key} is now set to {value} ({_type.__name__}) in overrides file.")
+
+    with settings.OVERRIDES_FILE.open("w") as f:
+        yaml.dump(overrides, f)
+
+
+def read(args: argparse.Namespace):
+    """Print the value of a specified settings variable.
+
+    If no key is provided, prints all available settings variables.
+    """
+    if not args.key:
+        for key, value in settings.settings.items():
+            print(f"{key}: {value} ({type(value).__name__})")
```

Comment:
Nit: maybe a `print_config_value` function would be nice in case we want to update the format of the output.